src/neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -91,6 +91,9 @@ private void OnMessage(Message msg)
                 case MessageCommand.Block:
                     OnInventoryReceived((Block)msg.Payload);
                     break;
+                case MessageCommand.BlockData:
+                    OnBlockDataReceived((Block)msg.Payload);
```

Comment:
What is the difference with `Block`?

Reply:
We are discussing the problem，Maybe we should delete it.

Reply:
Yes, we should delete it.